index.js

```diff
@@ -72,7 +76,7 @@ AssetsWebpackPlugin.prototype = {
         var assets
 
         if (self.options.entrypoints) {
-          assets = stats.entrypoints[chunkName].assets
+          assets = (self.options.includeFilesWithoutChunk && chunkName) ? stats.entrypoints[chunkName].assets : stats.assets
```

Comment:
I think putting `self.options.includeFilesWithoutChunk && chunkName` into a variable would help readability a lot.

Reply:
After testing, it turns out it isn't required to test for the `includeFilesWithoutChunk` option here. It's enough to check for the `chunkName` variable to see if we can access `stats.entrypoints` object or `stats.assets` for the unnamed chunk. Updated in 810b018